liquidity-module/index.md

```diff
@@ -90,18 +111,22 @@ Cosmos SDK's version is: Stargate v0.40.0 (or later)
 💿 Initializing the app...
 🙂 Created account "alice" with address "cosmos1qur6tvu7p4khtr5zzcx0uk5fq06hfk5xflw83e" with mnemonic: "tank film icon helmet myth devote velvet rib behind exhaust move pass endless combine bag congress pool bean shoulder issue trouble banner best nice"
 🙂 Created account "bob" with address "cosmos1w8w3t8rhv5zpvdscy9332wp4tugkg0ezskf95u" with mnemonic: "attract shoulder person upset dream category finish detect country track edge planet crack gloom soldier together hockey arena panel horn rapid zero common marriage"
-🙂 Created account "username" with address "cosmos1mr3ss57xexzg7j377vfd24d3vv3vy0e3mpuj6y" with mnemonic: "parent butter piece picnic north thumb knife denial toy silk juice diary cruise idle pink repair radar brisk decide sugar gap joke palm day"
-Genesis transaction written to "/Users/youruser/.myblockchaind/config/gentx/gentx-2e8a6a680b4f9adaecfafbc6ecad8b96ef8b9157.json"
+🙂 Created account "joy20" with address "cosmos1mr3ss57xexzg7j377vfd24d3vv3vy0e3mpuj6y" with mnemonic: "parent butter piece picnic north thumb knife denial toy silk juice diary cruise idle pink repair radar brisk decide sugar gap joke palm day"
```

Comment:
lets use a mock username (not "username")

Reply:
this is a good idea but only in the following example `"/Users/joy20/.myblockchaind..."`.

`Created account "joy20"...` would not be accurate since this mirrors what we have put into the `config.yml` file

```
accounts:
  - name: username
    coins: ["10000token", "50000000stake", "1000000000000mytoken"]
``` 

Or else we change `username` everywhere accordingly.